### ISSUE TITLE
function-whitespace-after may need to avoid function calls in quotes

### ISSUE THREAD

**Issue by joelpittet** [stylelint/stylelint#3145]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?
Coming from this Drupal core issue trying to enable this rule.

https://www.drupal.org/project/drupal/issues/2866815

Adding space inside quotes

```
-  content: " ("attr(lang)")";
+  content: " ("attr(lang) ")";
```

> Which rule, if any, is this issue related to?

`function-whitespace-after`

> What CSS is needed to reproduce this issue?

```css
.foo {
  content: " ("attr(lang)")";
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "function-whitespace-after": ["always"],
  }
}
```

> Which version of stylelint are you using?
 `7.13.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint "**/*.css"`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Not that I'm aware of.

> What did you expect to happen?

Quoted values would be ignored.

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
modules/ckeditor/css/plugins/language/ckeditor.language.css
 16:26  ✖  Expected whitespace after ")"   function-whitespace-after

themes/stable/css/ckeditor/plugins/language/ckeditor.language.css
 16:26  ✖  Expected whitespace after ")"   function-whitespace-after
```

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

The rule works as expected is asks to put space here, which is correct behaviour:

```css
.foo {
  content: " ("attr(lang)")";
  /*                    ↑     */
}
```

**Comment by joelpittet** [comment]
Sorry for bothering, I should have caught that but my eyes weren't parsing the quotes correctly. Thanks for the quick reply.
